swatch-producer-aws/src/main/java/com/redhat/swatch/processors/BillableUsageProcessor.java

```diff
@@ -78,13 +79,18 @@ public BillableUsageProcessor(
   @Incoming("tally-in")
   @Blocking
   public void process(BillableUsage billableUsage) {
-    if (log.isDebugEnabled()) {
-      log.debug("Picked up billable usage message {} to process", billableUsage);
-    }
+    log.debug("Picked up billable usage message {} to process", billableUsage);
     if (billableUsage == null) {
       log.warn("Skipping null billable usage: deserialization failure?");
       return;
     }
+    if (billableUsage.getOrgId() != null) {
+      MDC.put("OrgId", billableUsage.getOrgId());
+    }
+    if (billableUsage.getAccountNumber() != null) {
+      MDC.put("account_id", billableUsage.getAccountNumber());
+    }
```

Comment:
nitpick: can we please be consistent with casing (I prefer snake_casing personally)? So, OrgId -> org_id